README.md

```diff
@@ -131,53 +131,55 @@ controller_server:
       plugin: "mppi::Controller"
       time_steps: 15
       model_dt: 0.1
-      batch_size: 400
+      batch_size: 300
       vx_std: 0.2
       vy_std: 0.2
       wz_std: 1.0
       vx_max: 0.5
       vy_max: 0.5
       wz_max: 1.3
-      iteration_count: 2
-      prune_distance: 1.5
+      iteration_count: 1
```

Comment:
For 20hz i didn't notice any difference with 2. Probably we could use 3+ and bigger batch if optimize reference critic / make eigen port

Reply:
We're also working mostly in simulation, I think it would be good to have at least 1 additional refinement iteration (2) even if we don't notice the difference in sim